molparse/system.py

```diff
@@ -19,6 +19,7 @@ def __init__(self, name: str):
         self.description = None
         self.remarks = []
         self._header_data = []
+        self.ssbonds=[]
```

Comment:
This is optional but because the user should probably not modify this directly I think it's best to name the attribute `_ssbonds` and create a `@property` for `ssbonds`.